base/set.jl

```diff
@@ -196,15 +196,26 @@ end
 Removes recurrences of items and returns the modified collection
 """
 function unique!(itr)
-	seen = Set{eltype(itr)}()
-	for (i, x) in enumerate(itr)
-		if x in seen
-			deleteat!(itr, i)
-		else
-			push!(seen, x)
-		end
-	end
-	return itr
+   T = eltype(itr)
+   seen = Set{T}()
+   duplicateIndex = Vector{Int64}()
```

Comment:
The style guide in CONTRIBUTING.md says not to use camelCase. You can just call this e.g. `dups`.